apps/test/unit/lib/ui/SchoolInfoInterstitialTest.jsx

```diff
@@ -50,16 +50,59 @@ describe('SchoolInfoInterstitial', () => {
             />
           </div>
           <div>
+            <Button
+              text={i18n.dismiss()}
+              onClick={wrapper
+                .find(Button)
+                .first()
+                .prop('onClick')}
+            />
             <Button
               text={i18n.save()}
-              onClick={wrapper.find(Button).prop('onClick')}
+              onClick={wrapper
+                .find(Button)
+                .last()
+                .prop('onClick')}
             />
           </div>
         </div>
       </BaseDialog>
     );
   });
 
+  it('closes the school info interstitial modal when the dismiss button is clicked', () => {
+    const wrapper = shallow(
```

Comment:
does `mount` instead of `shallow` here fix the problem with the spy? https://github.com/airbnb/enzyme/issues/944 seems to indicate it will

Reply:
Yes, it does ---> wrapper.instance().forceUpdate();